### DISCUSSION TITLE
Welcome to Community Discussions!

### DISCUSSION THREAD

**Discussion by MorpheusTNT** [Titannet-dao/Community#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cr0812** [comment]
I am a node operator from China, currently only running in the operation node, I would like to ask when the test network will end

**Reply by MorpheusTNT** [reply]
The first test network is scheduled to end in April

**Reply by MaryTitan** [reply]
The first round of the testnet, which is the Huygens Testnet, ends around mid of April. We will release announcement about ending time about one week ahead. Please follow us on [Twitter](https://twitter.com/Titannet_dao) for the latest updates. Thank you.

**Comment by bXoooo** [comment]
很高兴能够参加这个项目。

**Reply by MorpheusTNT** [reply]
欢迎共建

**Reply by MaryTitan** [reply]
We are happy to have you with us. :)

**Comment by lyj8964** [comment]
One device can be bound to several accounts

**Reply by MaryTitan** [reply]
Each Titan account can link to multiple devices. You can run up to 5 devices/nodes under one IP.

**Comment by evilivef** [comment]
windows exe 運行後無法綁定身份碼，煩請提供解決思路，謝謝

**Reply by MaryTitan** [reply]
Please join our TG group for bug report and customer service. Thank you. 
(https://t.me/titannet_dao)

**Comment by thompsonng** [comment]
I am a node runner in New Zealand
Will the storage limit affect the the rewards? I am using 64 GBs. 
How many factors affect the rewards daily?

**Reply by MaryTitan** [reply]
Hi there! Thank you for your question. Yes, your storage space is one of the factors that influence your rewards. Please refer to our Gitbook for reward details. (https://titannet.gitbook.io/titan-network-en)

**Comment by xxcms** [comment]
运行节点的奖励在主网上线后会有用吗？

**Reply by MaryTitan** [reply]
Of course! The rewards you earn in testnets will be converted in a certain ratio before mainnet launch.

**Reply by Maus313** [reply]
比例方便透露吗？

**Comment by tianxiaguixin666** [comment]
我在修改共享存储空间后，显示的titan可用空间依旧是2GB，我无法使用telegram，希望你在这里解答，谢谢！

**Reply by MaryTitan** [reply]
I have connected with you on Discord. Cheers.

**Comment by Amran3321** [comment]
Ini komentar dari amran3321, saya baru belajar kontribusi GitHub.
